app/controllers/api/v1/search_controller.rb

```diff
@@ -294,11 +294,24 @@ def index
         # perform TDR search, if enabled, and there are facets/terms provided by user
         if User.feature_flag_for_instance(current_api_user, 'cross_dataset_search_backend') && (@facets.present? || @term_list.present?)
           @tdr_results = self.class.get_tdr_results(selected_facets: @facets, terms: @term_list)
+          
+          if (@facets.present?)
```

Comment:
Redundant parentheses - Ruby doesn't require them for `if/else` blocks.
```suggestion
          if @facets.present?
```